test/x86/avx512f.c

```diff
@@ -1309,6 +1534,214 @@ test_simde_mm512_setone_si512(const MunitParameter params[], void* data) {
+                              UINT64_C(0x9f43b81b73f10ecc), UINT64_C(0x5cf361258c69074e),
+                              UINT64_C(0xff40b7e681a97da7), UINT64_C(0xa0db6e89c3bba587),
+                              UINT64_C(0x1ef0d82a3032940e), UINT64_C(0x6e6129fbad8436eb)),
+      UINT32_C(26),
+      simde_x_mm512_set_epu64(UINT64_C(0x0000000ef9c9533d), UINT64_C(0x0000000f8c191bbb),
+                              UINT64_C(0x00000027d0ee06dc), UINT64_C(0x000000173cd84963),
+                              UINT64_C(0x0000003fd02df9a0), UINT64_C(0x0000002836dba270),
+                              UINT64_C(0x00000007bc360a8c), UINT64_C(0x0000001b984a7eeb)) },
+    { simde_x_mm512_set_epu64(UINT64_C(0x8c60b9ee3084e68a), UINT64_C(0x7df27a77163f440e),
+                              UINT64_C(0x1c8e2ce2cbcdb26b), UINT64_C(0x9a875b341e54f334),
+                              UINT64_C(0xadd74f036b2fd29e), UINT64_C(0x4df18e156438f1fb),
+                              UINT64_C(0x006d02efd4f52b83), UINT64_C(0xc04924106403aeb8)),
+      UINT32_C(27),
+      simde_x_mm512_set_epu64(UINT64_C(0x000000118c173dc6), UINT64_C(0x0000000fbe4f4ee2),
+                              UINT64_C(0x0000000391c59c59), UINT64_C(0x0000001350eb6683),
+                              UINT64_C(0x00000015bae9e06d), UINT64_C(0x00000009be31c2ac),
+                              UINT64_C(0x000000000da05dfa), UINT64_C(0x000000180924820c)) },
+    { simde_x_mm512_set_epu64(UINT64_C(0xb37b4d212858a1f2), UINT64_C(0x332c5a15c349f1f8),
+                              UINT64_C(0x89c2363c78a22d75), UINT64_C(0x7a88d63e31490779),
+                              UINT64_C(0x49a6fa27e84df7f6), UINT64_C(0x7a2a8b04c05b79dc),
+                              UINT64_C(0x2cb04d0b912c9a91), UINT64_C(0x38ff445c61499eb7)),
+      UINT32_C(5),
+      simde_x_mm512_set_epu64(UINT64_C(0x059bda690942c50f), UINT64_C(0x019962d0ae1a4f8f),
+                              UINT64_C(0x044e11b1e3c5116b), UINT64_C(0x03d446b1f18a483b),
+                              UINT64_C(0x024d37d13f426fbf), UINT64_C(0x03d154582602dbce),
+                              UINT64_C(0x016582685c8964d4), UINT64_C(0x01c7fa22e30a4cf5)) },
+    { simde_x_mm512_set_epu64(UINT64_C(0xef4891f108bd3572), UINT64_C(0x70ed11eef2e11aa0),
+                              UINT64_C(0xef3984671f62c16d), UINT64_C(0x00cdcea30b923238),
+                              UINT64_C(0x44ef5f23e94bd07e), UINT64_C(0x61a977d25ae9865b),
+                              UINT64_C(0xbe912feb87b529cc), UINT64_C(0x6a55790501f02cd4)),
+      UINT32_C(1),
+      simde_x_mm512_set_epu64(UINT64_C(0x77a448f8845e9ab9), UINT64_C(0x387688f779708d50),
+                              UINT64_C(0x779cc2338fb160b6), UINT64_C(0x0066e75185c9191c),
+                              UINT64_C(0x2277af91f4a5e83f), UINT64_C(0x30d4bbe92d74c32d),
+                              UINT64_C(0x5f4897f5c3da94e6), UINT64_C(0x352abc8280f8166a)) },
```

Comment:
Hex can be nicer to look at for bitwise operations, but all the assertion functions (like `simde_assert_m512i_u64`) print errors in decimal.  Using hex here makes it hard to tell which case is failing.

Reply:
OK I'll change.
I deliberately went to the trouble of using hex for most bitwise operations as it is more understandable for humans :-(

Reply:
Yeah, I agree, it's easier to look at for that.  If you feel strongly about it you could always add macros like `simde_assert_m128i_i32x` (note the x suffix) for a hex version.  I'd accept a patch, but I'm not going to bother writing it myself ;)